runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/SdkHarnessClient.java

```diff
@@ -50,7 +50,7 @@
  * <p>This provides a Java-friendly wrapper around {@link FnApiControlClient} and {@link
  * CloseableFnDataReceiver}, which handle lower-level gRPC message wrangling.
  */
-public class SdkHarnessClient {
+public class SdkHarnessClient implements AutoCloseable {
```

Comment:
Just for my edification: if I am authoring a runner and I currently interact directly with a `FnApiControlClient` that I got from a `FnApiControlClientPoolService`, should I instead be trying to use an `SdkHarnessClient` that I got from an `SdkHarnessClientPoolService`?

Reply:
Yes. `FnApiControlClientPoolService` shouldn't really be visible outside of the `control` package